nixos/modules/services/web-apps/plausible.nix

```diff
@@ -0,0 +1,270 @@
+{ lib, pkgs, config, ... }:
+
+with lib;
+
+let
+  cfg = config.services.plausible;
+
+  envSecrets = ''
+    export ADMIN_USER_PWD="$(<${cfg.adminUser.passwordFile})"
+    export SECRET_KEY_BASE="$(<${cfg.server.secretKeybaseFile})"
+    export RELEASE_TMP=/var/lib/plausible/tmp
```

Comment:
not sure this one needs to be a secret. You could potentially just add it to the environment